### PR TITLE
updated companies for urls : Issue:116

### PR DESCRIPTION
**Pull request by tolgaouz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -306,6 +306,12 @@ <h2 class="font-weight-bold">
                       {{ company.description }}
                     </p>
                     
+                    <!-- Link -->
+                    {% if company.url %}
+                      <a href={{ company.url.href }} name={{ company.url.name }} style="color: {{ company.color }}">Read more</a>
```
Comment (davidgatti) [inline index.html:311]: Mr inpatient, stare at this line and figure out what is wrong ;)

Comment (davidgatti) [reply index.html]: No, what is the point of this line? `name={{ company.url.name }}` who is going to see the name of the URL? the computer? Google? what about humans? Maybe they should see it? ;)

Comment (tolgaouz) [reply index.html]: Ah! Okay, when you said name I thought you were referring to name attribute of the tag. Fixing it

Comment (davidgatti) [reply index.html]: aaaaa @tolgaouz :D

**Pull request merged**